MekHQ/src/mekhq/service/MassRepairService.java

```diff
@@ -889,6 +1192,243 @@ public int compare(Person tech1, Person tech2) {
 		}
 	}
 
+	public static class MassRepairPartAction {
+		public interface STATUS {
+			public static final int REPAIRED = 0;
+			public static final int MAX_SKILL_REACHED = 1;
+			public static final int MRO_DISABLED = 2;
+			public static final int NO_TECHS = 3;
+		}
```

Comment:
Seems like this should be an `enum`.  That way you can limit the `status` argument in your constructor to just these values.

Reply:
Sure. We can turn these into enums. Mostly we only gain type-safety and protection from the random data, but I'm fine to change it.